File: wwwroot/js/game.js
const connection = new signalR.HubConnectionBuilder()
    .withUrl("/gamehub")
    .build();

connection.on("ReceiveMove", (fromX, fromY, toX, toY) => {
    // Update the game board based on the move
});

async function makeMove(fromX, fromY, toX, toY) {
    await connection.invoke("SendMove", fromX, fromY, toX, toY);
}

connection.start().catch(err => console.error(err.toString()));

File: Pages/Models/Game.cs
public class Game
{
    private const string V = "Player1";

    public string[,] Board { get; set; }
    public string CurrentPlayer { get; set; }
    public bool IsGameOver { get; set; }

    public Game()
    {
        Board = new string[5, 5];
        CurrentPlayer = V;
        IsGameOver = false;
    }
}